packages/base-contract/CHANGELOG.json

```diff
@@ -1,4 +1,13 @@
 [
+    {
+        "version": "6.1.0",
+        "changes": [
+            {
+                "note": "Catch empty reverts on live networks.",
```

Comment:
Nit: shouldn't this be just "_Support_ catching..." ?  Since you just added a function but never call it, within `base-contract` itself.

Reply:
tru dat